src/scene/lightmapper.js

```diff
@@ -412,15 +411,12 @@ Object.assign(Lightmapper.prototype, {
         // Create pseudo-camera
         if (!lmCamera) {
             lmCamera = new Camera();
-            lmCamera._node = new GraphNode();
-            lmCamera.clearColor[0] = 0;
-            lmCamera.clearColor[1] = 0;
-            lmCamera.clearColor[2] = 0;
-            lmCamera.clearColor[3] = 0;
-            lmCamera.clearDepth = 1;
```

Comment:
We don't clear the depthBuffer in the previous version of the lightmapper. Bug or designed behaviour?

Reply:
Check out the next line. `clearFlags` used to be set to just `CLEARFLAG_COLOR`. So previously, the depth bugger was never cleared before. So no change in behavior.